src/components/clusterprovider/minikube/minikubeclusterprovider.ts

```diff
@@ -0,0 +1,224 @@
+import * as restify from 'restify';
+import * as portfinder from 'portfinder';
+import * as clusterproviderregistry from '../clusterproviderregistry';
+import { styles, formStyles, waitScript, ActionResult, Diagnostic, fromShellExitCodeOnly } from '../../../wizard';
+import { propagationFields, formPage } from '../common/form';
+import { refreshExplorer } from '../common/explorer';
+import { Errorable, succeeded, failed, Failed, Succeeded } from '../../../errorable';
+import { Shell } from '../../../shell';
+
+export interface Context {
+    readonly shell: Shell;
+    createExec: Promise<any>;
```

Comment:
I may be missing something due to reading this in a browser rather than working through it in a proper editor, but it looks like `createExec` is only ever set, never read.  So does it truly need to be part of the context?  Perhaps an earlier iteration checked whether the promise had completed but that has now been replaced by a `minikube status` call or something?  Sorry if I'm missing something...

Reply:
fixed.